### DISCUSSION TITLE
数据源求推荐

### DISCUSSION THREAD

**Discussion by dukeluo** [dukeluo/wai#85] [Ideas]
友友们，数据源求推荐，大致要求：
- 有趣实用（更有动力刷新标签页，停下手中活，多扭几下脖子）
- 纯文本（利于展示）
- 有一定数量（不至于重复）
- 要是能支持英文那就最最好了（走出国门，冲向世界）

欢迎大家提供思路！

**Comment by cike-567** [comment]
考虑过用 RSS 订阅吗？

**Reply by dukeluo** [reply]
那个 RSS 订阅，我看看先👀

**Reply by cike-567** [reply]
https://docs.rsshub.app/
https://www.v2ex.com/t/936952
https://github.com/weekend-project-space/top-rss-list
还有很多的 RSS 订阅源，可以在网络上找到。

**Reply by dukeluo** [reply]
如果展示一般意义上的 RSS 源的内容，展示区域太小，展示不完全，也不美观

**Comment by RainMelody0** [comment]
要不要试试诗歌集？

**Reply by dukeluo** [reply]
像「几枝」那样吗

**Reply by RainMelody0** [reply]
推荐《哀江南赋》，字数多又优美，古代骈文第一，每天一句起码可以半年。

**Reply by RainMelody0** [reply]
btw，我觉得添加的文字内容要从你的个人美学出发，成长性会比较好
